plugins/generator-1.18.2/forge-1.18.2/procedures/get_command_parameter_string.java.ftl

```diff
@@ -0,0 +1,14 @@
+<#if field$type == "string">

```

Comment:
What is the difference between message and string?

Reply:
Message is an argument typically used in commands such as `/say`, while String is an argument with a single word, a quotable text `"like this"` and another work I do not remember, but that is not implemented. I did not add the message argument for the moment and a few others as this PR is already pretty big and I just wanted to add main arguments. I will open another PR just after this one is merged to add final (and minor) blocks).

Reply:
Hmm, so this depends if the argument is single_word or not, this is a bit confusing then as user can not associate these two

Reply:
Could not this be done automatically so user does not have to be confused by this?